### DISCUSSION TITLE
Is ruby-2.2's `gem` "somehow broken"?

### DISCUSSION THREAD

**Discussion by stdedos** [ruby/setup-ruby#652] [Q&A] [answered]
_I have read the Issue bug disclaimer, hence the discussion_

Action: https://github.com/stdedos/interception/actions/runs/11141393880/job/30962174539#step:3:26

Search:

- https://github.com/ruby/setup-ruby/issues?q=sort%3Aupdated-desc+ruby+2.2.10
  - https://github.com/ruby/setup-ruby/issues/496
- https://github.com/ruby/setup-ruby/issues?q=sort%3Aupdated-desc+Segmentation+fault6
  - https://github.com/ruby/setup-ruby/issues/150

But barely the https://github.com/ruby/setup-ruby/issues/496 would fit (as far as I can read)

**Comment by dentarg** [comment, marked as answer]
Read https://github.com/ruby/setup-ruby/issues/496#issuecomment-1520667671
